WordPress/Classes/Extensions/UIImageView+Gravatar.swift

```diff
@@ -118,6 +122,56 @@ extension UIImageView {
         sessionConfiguration?.urlCache?.removeAllCachedResponses()
     }
 
+    /// Sets the provided icon as content
+    ///
+    /// Parameters:
+    ///  - siteIcon: string description of the site's icon
+    ///
+    @objc func setImageWithSiteIcon(_ siteIcon: String) {
+        let blavatarDefaultImage = UIImage(named: UIImageView.BlavatarDefault)
+        self.setImageWithSiteIcon(siteIcon, placeholderImage: blavatarDefaultImage)
+    }
+
+    /// Sets the provided icon as content
+    ///
+    /// Parameters:
+    ///  - siteIcon: string description of the site's icon
+    ///  - placeholderImage: the image to be used as a placeholder
+    ///
+    @objc func setImageWithSiteIcon(_ siteIcon: String, placeholderImage: UIImage?) {
+        let url = URLWithSiteIcon(siteIcon)
+
+        if url != nil {
```

Comment:
You may replace this line, and the one above, with something like this:

```
guard let url = URLWithSiteIcon(siteIcon) else {
   return
}
```

That's (way better!) because you both get to reduce indentation, and avoid the implicitly unwrapped optional.